pkg/controller/deployment/operator_log_level_test.go

```diff
@@ -0,0 +1,48 @@
+package deployment
+
+import (
+	"fmt"
+	"testing"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"k8s.io/klog/v2"
+)
+
+func TestSetOperatorLogLevel(t *testing.T) {
+	testCases := []struct {
+		logLevel              operatorv1.LogLevel
+		desiredVerbosityLevel int
+	}{
+		{
+			logLevel:              operatorv1.Normal,
+			desiredVerbosityLevel: 2,
```

Comment:
Can you add negative cases as well?

Reply:
https://github.com/openshift/cert-manager-operator/pull/111#discussion_r1169705175